azurerm/internal/services/loganalytics/tests/log_analytics_saved_search_resource_test.go

```diff
@@ -205,3 +224,35 @@ resource "azurerm_log_analytics_saved_search" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger, data.RandomInteger)
 }
+
+func testAccAzureRMLogAnalyticsSavedSearch_withTag(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_log_analytics_workspace" "test" {
+  name                = "acctestLAW-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku                 = "PerGB2018"
+}
+
+resource "azurerm_log_analytics_saved_search" "test" {
+  name                       = "acctestLASS-%d"
+  log_analytics_workspace_id = azurerm_log_analytics_workspace.test.id
+
+  category     = "Saved Search Test Category"
+  display_name = "Create or Update Saved Search Test"
+  query        = "Heartbeat | summarize Count() by Computer | take a"
+  tags = {
+    "Environment" = "test"
```

Comment:
we should always ensure capitals work for both key and value
```suggestion
    "Environment" = "Test"
```